### DISCUSSION TITLE
Code formatting option?

### DISCUSSION THREAD

**Discussion by kkweon** [EvgSkv/logica#401] [General]
I keep fighting my editor doing the wrong formatting/indent while trying to be smart.

What code formatting tool is available for logica? If none, should we have one? I see Prettier can be extended for other language.

https://prettier.io/docs/en/plugins.html

**Comment by EvgSkv** [comment]
Hello @kkweon !
Depending on machine and task I'm using vscode, sublime or vim. None of them has formatting for Logica, but they are not being too clever on me.
Yes, having formatter would probably be nice :-)
